executor/common.h

```diff
@@ -490,6 +490,9 @@ static void loop(void)
 #if SYZ_HAVE_CLOSE_FDS
 	close_fds();
 #endif
+#if SYZ_EXECUTOR || __NR_syz_usb_connect
+	usb_reset();
```

Comment:
usb_reset() is only defined on linux, won't this cause build failures on all other OSes? Or what am I missing?

Reply:
I think this code is never used in SYZ_EXECUTOR mode. If so the SYZ_EXECUTOR is not need and is confusing.

Reply:
Removed `usb_reset()`.